code/_onclick/item_attack.dm

```diff
@@ -234,7 +234,8 @@
 		user.client.give_award(/datum/award/achievement/misc/selfouch, user)
 
 	user.do_attack_animation(target_mob)
-	target_mob.attacked_by(src, user)
+	if(!target_mob.attacked_by(src, user))
```

Comment:
You should make this proc return `TRUE` if force <= 0 rather than `FALSE`, it'll fix CI, and makes sense I guess

Reply:
done